daemon/src/taker.rs

```diff
@@ -179,10 +179,21 @@ async fn main() -> Result<()> {
     let maker_identity = Identity::new(opts.maker_id);
 
     let seed = Seed::initialize(&data_dir.join("taker_seed")).await?;
+    let (_, identity_sk) = seed.derive_identity();
+
+    // If the wallet seed does not exist, create one using the seed
+    if !&data_dir.join("taker_wallet_seed").exists() {
+        tokio::fs::copy(
+            &data_dir.join("taker_seed"),
+            &data_dir.join("taker_wallet_seed"),
+        )
+        .await?;
+    }
+
+    let wallet_seed = Seed::initialize(&data_dir.join("taker_wallet_seed")).await?;
 
     let bitcoin_network = opts.network.bitcoin_network();
-    let ext_priv_key = seed.derive_extended_priv_key(bitcoin_network)?;
-    let (_, identity_sk) = seed.derive_identity();
+    let ext_priv_key = wallet_seed.derive_extended_priv_key(bitcoin_network)?;
```

Comment:
I think we should change the interface of `wallet::Actor` to be initialized by a `Mnemonic`. That will clean up this back and forth here.